Sources/Orbit/WIP/PasswordStrengthIndicator.swift

```diff
@@ -0,0 +1,107 @@
+import SwiftUI
+
+public enum PasswordStrength {
```

Comment:
Wouldn't it make sense to nest this inside the view?

Reply:
Agree. It is usually driven by whether the View is generic or not (even though rarely, this sometimes changes from non-generic->generic)

Reply:
As `PasswordStrength` is a param of new `InputField.init` it will stay public and therefore outside of PasswordStrengthIndicator which is still internal

Reply:
That, or the indicator component could be a `public` component as well so that client can use it to construct similar components.

Reply:
Missed that comment, will update